CppCoreGuidelines.md

```diff
@@ -20320,20 +20320,20 @@ and errors (when we didn't deal correctly with semi-constructed objects consiste
 
     class Picture
     {
-        size_t mx;
-        size_t my;
+        ptrdiff_t mx;
+        ptrdiff_t my;
```

Comment:
why not just `int` then?

Reply:
To handle really large sizes appropriately. I guess it should be consistent though, not sure whether to change the above to prrdiff_t or this to int

Reply:
`ptrdiff_t` is less than `size_t`.

https://en.cppreference.com/w/cpp/types/ptrdiff_t